backend/api/Areas/Documents/DocumentController.cs

```diff
@@ -38,152 +37,135 @@ public class DocumentController : ControllerBase
         /// Creates a new instance of a ErrorController class.
         /// </summary>
         /// <param name="documentService"></param>
-        /// <param name="documentSyncService"></param>
         /// <param name="mapper"></param>
-        public DocumentController(IDocumentService documentService, IDocumentSyncService documentSyncService, IMapper mapper)
+        public DocumentController(IDocumentService documentService, IMapper mapper)
         {
             _documentService = documentService;
-            _documentSyncService = documentSyncService;
             _mapper = mapper;
         }
         #endregion
 
         #region Endpoints
 
         /// <summary>
-        /// Retrieves the list of document types.
+        /// Get the document types.
         /// </summary>
+        /// <returns></returns>
         [HttpGet("types")]
-        [HasPermission(Permissions.PropertyAdd)]
-        [ProducesResponseType(typeof(ExternalResult<QueryResult<DocumentType>>), 200)]
-        [SwaggerOperation(Tags = new[] { "documents" })]
+        //[HasPermission(Permissions.DocumentView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(List<Concepts.DocumentTypeModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "types" })]
         public IActionResult GetDocumentTypes()
         {
-            var result = _documentService.GetStorageDocumentTypes();
-            return new JsonResult(result);
+            var documentTypes = _documentService.GetPimsDocumentTypes();
+            var mappedDocumentTypes = _mapper.Map<List<Concepts.DocumentTypeModel>>(documentTypes);
+            return new JsonResult(mappedDocumentTypes);
         }
 
         /// <summary>
-        /// Retrieves a list of documents.
+        /// Gets a collection of documents for the specified type and owner id.
         /// </summary>
-        [HttpGet]
-        [HasPermission(Permissions.PropertyAdd)]
+        /// <param name="type">Used to identify document type.</param>
+        /// <param name="entityId">Used to identify document's parent entity.</param>
+        /// <returns></returns>
+        [HttpGet("{relationshipType}/{parentId:long}")]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(ExternalResult<QueryResult<DocumentDetail>>), 200)]
-        [SwaggerOperation(Tags = new[] { "documents" })]
-        public IActionResult GetDocumentList()
+        //[HasPermission(Permissions.NoteView)]
+        [ProducesResponseType(typeof(IList<DocumentRelationshipModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "document" })]
```

Comment:
should this be storage-documents?

Reply:
No since this actually returns pims related documents. I am calling storage-documents the ones comming from the mayan storage